smolyay/basis.py

```diff
@@ -1,8 +1,15 @@
 import abc
+import collections.abc
 
 import numpy
 import scipy.special
 
+from smolyay._growth_method import (
+    ClenshawCurtisSlowExponentialGrowthMixin,
+    ClenshawCurtisExponentialGrowthMixin,
+    TrigonometricExponentialGrowthMixin,
+)
```

Comment:
Try not to pollute your namespace too much
```suggestion
from smolyay import _growth
```